CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Release Date: 2021-??-?? Valhalla 3.1.2
 * **Removed**
 * **Bug Fix**
+   * FIXED: Fix reclassification for "shorter" ferries and rail ferries (for Chunnel routing issues) [#3038](https://github.com/valhalla/valhalla/pull/3038)
```

Comment:
can you move this to the bottom of the bug fix list? for whatever reason we've been adding them to the bottom of each of the respective lists so that the oldest is at the top. i guess so that when you read a releases change log its in chronological order from oldest to newest changes.

Reply:
Done